cmake/modules/RootCPack.cmake

```diff
@@ -29,12 +29,26 @@ string(REGEX REPLACE "^([0-9]+).*$" "\\1" CXX_MAJOR ${CMAKE_CXX_COMPILER_VERSION
 string(REGEX REPLACE "^([0-9]+)\\.([0-9]+).*$" "\\2" CXX_MINOR ${CMAKE_CXX_COMPILER_VERSION})
 
 #---Resource Files-----------------------------------------------------------------------------------
-configure_file(README.md README.md COPYONLY)
 configure_file(LICENSE LICENSE.txt COPYONLY)
 configure_file(LGPL2_1.txt LGPL2_1.txt COPYONLY)
-set(CPACK_PACKAGE_DESCRIPTION_FILE "${CMAKE_BINARY_DIR}/README.md")
 set(CPACK_RESOURCE_FILE_LICENSE "${CMAKE_BINARY_DIR}/LICENSE.txt")
-set(CPACK_RESOURCE_FILE_README "${CMAKE_BINARY_DIR}/README.md")
+if (APPLE)
+  find_program(CONVERTER textutil)
+  if (NOT CONVERTER)
+    message(FATAL_ERROR "textutil executable not found")
+  endif()
+  execute_process(COMMAND ${CONVERTER} -convert html README.md)
+else()
+  find_program(CONVERTER pandoc)
+  if (NOT CONVERTER)
+    message(FATAL_ERROR "pandoc executable not found")
+  endif()
+  execute_process(COMMAND ${CONVERTER} README.md -f markdown -t html -s -o README.html)
+endif()
+configure_file(README.html README.html COPYONLY)
```

Comment:
I don't understand the need for this. Either `README.html` gets generated by `execute_process` in the source directory (that should be fixed) or this line shouldn't be needed? Could you explain, please?

Reply:
Thanks for pointing that out. It should be fixed now.